recipe/meta.yaml

```diff
@@ -13,7 +13,12 @@ source:
 
 build:
   number: 0
-  script: python -m pip install --no-deps --ignore-installed .
+  script: |
+    python -m pip install --no-deps --ignore-installed .
+    # This is a hack to get the custom css file in the right place. Remove once
+    # https://gitlab.com/moorepants/skijumpdesign/issues/44 is addressed
+    # properly.
+    mv {{ environ["PREFIX"] }}/static {{ environ["SP_DIR"] }}/
```

Comment:
Should be able to add it to `MANIFEST.in`.

Reply:
It is in the MANIFEST.in. See this SO question for the details:

https://stackoverflow.com/questions/50160009/how-can-i-package-a-css-file-with-a-single-module-flask-application-with-setupto